apps/frontend/src/components/atoms/Editor.tsx

```diff
@@ -112,9 +112,9 @@ const TextEditor: React.FC<Props> = ({ editor, minH = "70vh" }) => {
       )}{" "}
       <EditorContent
         // eslint-disable-next-line tailwindcss/no-contradicting-classname
-        className={`[&>.ProseMirror.tiptap] w-full break-words break-all`}
-        // style={{ minHeight: minH }}
-        style={{ minHeight: "20vh" }}
+        className={`[&>.ProseMirror.tiptap]:text-foreground w-full break-words break-all`}
+        style={{ minHeight: minH }}
```

Comment:
here you need to make the minHeight half,

right now minH is goiving 40vh height ,

but we need 20vh height

can you make the change

Reply:
![image](https://github.com/user-attachments/assets/5530b34f-cd97-4f5d-bd09-11d05f4f0fc5)
the minH of the editor on todays page is 40vh because its setted in TodayTextArea component.
the editor component is beeing used in a lot of pages, so i need the height to be changeable. this is why the minH is a component paramter.
if you want the editor on todays page to be 20vh we need to change the TodayTextArea component, not the editor component.

Reply:
then please do that

Reply:
I just made a commit changing the minH of the TodayTextArea component from 40vh to 20vh